.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = library/securimage
 	url = https://github.com/mantisbt/securimage.git
 	branch = mantis
+[submodule "library/parsedown"]
+	path = library/parsedown
+	url = https://github.com/erusev/parsedown.git
+	branch = mantisbt
```

Comment:
This is incorrect as there is no `mantisbt` branch at https://github.com/erusev/parsedown.

What we normally do for submodules is first fork the repo to mantisbt.org, and create a mantis branch for any specific changes we might have vs the original library. Since you lack privileges to do this yourself, I'll do it for you. Until that's done, you should specify a valid branch in the upstream repository.